app/services/create_new_device_alert.rb

```diff
@@ -12,11 +12,17 @@ def perform(now)
       emails_sent += 1 if expire_sign_in_notification_timeframe_and_send_alert(user)
     end
 
+    analytics.create_new_device_alert_job_emails_sent(count: emails_sent)
+
     emails_sent
   end
 
   private
 
+  def analytics
+    @analytics ||= Analytics.new(user: AnonymousUser.new, request: nil, sp: nil, session: {})
```

Comment:
this is logging across all users? or for each user?

Reply:
The job runs independently of users (can dispatch emails for multiple users). 

This was copied from how it's done in `AccountReset::GrantRequestsAndSendEmails`:

https://github.com/18F/identity-idp/blob/708bd6b1abad0f187ddfe9980ad97c752e284bc8/app/services/account_reset/grant_requests_and_send_emails.rb#L23-L30